framework/include/updater.hpp

```diff
@@ -126,27 +136,33 @@ namespace gvk
-		 *   - destroying_graphics_pipeline_event ...... Triggered before an old, outdated graphics pipeline is destroyed.    Example: `gvk::destroying_graphics_pipeline_event()`
-		 *   - destroying_compute_pipeline_event ....... Triggered before an old, outdated compute pipeline is destroyed.     Example: `gvk::destroying_compute_pipeline_event()`
-		 *   - destroying_ray_tracing_pipeline_event ... Triggered before an old, outdated ray tracing pipeline is destroyed. Example: `gvk::destroying_ray_tracing_pipeline_event()`
-		 *   - destroying_image_event .................. Triggered before an old, outdated image is destroyed.                Example: `gvk::destroying_image_event()`
-		 *   - destroying_image_view_event ............. Triggered before an old, outdated image view is destroyed.           Example: `gvk::destroying_image_view_event()`
-		 * 
+		 *	 - files_changed_event ............................	Triggered after files have changed on disk.                          Example: `gvk::shader_files_changed_event(mPipeline)`
+		 *   - swapchain_changed_event ........................	Triggered after the swapchain has changed.                           Example: `gvk::swapchain_changed_event(gvk::context().main_window())`
+		 *   - swapchain_resized_event ........................	Triggered after the swapchain has been resized.                      Example: `gvk::swapchain_resized_event(gvk::context().main_window())`
+		 *   - swapchain_format_changed_event .................	Triggered after the swapchain format has changed.                    Example: `gvk::swapchain_format_changed_event(gvk::context().main_window())`
+		 *   - concurrent_frames_count_changed_event...........	Triggered after window #framesInFlight has changed.                  Example: `gvk::concurrent_frames_count_changed_event(gvk::context().main_window())`
+		 *   - swapchain_additional_attachments_changed_event..	Triggered after window additional attachments have changed.          Example: `gvk::swapchain_additional_attachments_changed_event(gvk::context().main_window())`
+		 *   - destroying_graphics_pipeline_event .............	Triggered before an old, outdated graphics pipeline is destroyed.    Example: `gvk::destroying_graphics_pipeline_event()`
+		 *   - destroying_compute_pipeline_event ..............	Triggered before an old, outdated compute pipeline is destroyed.     Example: `gvk::destroying_compute_pipeline_event()`
+		 *   - destroying_ray_tracing_pipeline_event ..........	Triggered before an old, outdated ray tracing pipeline is destroyed. Example: `gvk::destroying_ray_tracing_pipeline_event()`
+		 *   - destroying_image_event .........................	Triggered before an old, outdated image is destroyed.                Example: `gvk::destroying_image_event()`
+		 *   - destroying_image_view_event .................... Triggered before an old, outdated image view is destroyed.           Example: `gvk::destroying_image_view_event()`
+		 *
```

Comment:
Hmm, formatting probably only looks off here on GitHub. Unsure how we should best handle this in the future. Maybe spaces are better suited for formatting after all...

Reply:
replaced the instances I could think of with spaces.